docs/nut-command/setup/setup-deploy.md

```diff
@@ -5,20 +5,30 @@ level: intermediate
 setup:deploy
 ============
 
-Nut's `setup:deploy` is a simple tool to deploy a build of the current site 
+<p class="note"><strong>Note:</strong> The command is provided by a separate
+package called <code>bolt/simple-deploy</code> that is installed by default
```

Comment:
"<code>bolt/simple-deploy</code>, that" needs a comma, right?

Reply:
Nope, _"that is installed by default"_ is a restrictive clause in this case.

Reply:
Alright, then! 👍